docs/user-guide/docker-volume-cli.md

```diff
@@ -127,7 +137,7 @@ docker volume inspect MyVolume
     }
 ]
 ```
-Note: For disk formats zeroedthick and zeroedthick, the allocated size would be total size plus the size of replicas.
+Note: For disk formats zeroedthick and thin, the allocated size would be total size plus the size of replicas.
 
 ## Docker Compose
```

Comment:
Let's keep only docker volume related commands here.
Let's move Docker compose, service, stack etc commands to separate section - **Docker API compatibility**  that showcases how to use non-volume related docker commands in conjunction with vDVS.
We already have a separate section for Docker Swarm